examples/README.rst

```diff
@@ -1,4 +1,9 @@
 ===============
 Gallery
 ===============
-Examples using pylops_mpi
+Here is a collection of examples that demonstrate the usage of PyLops-MPI operators and utilities.
+
+.. note::
+    All examples are currently run on a single rank when building the documentation.
+    However, we test their correctness off-line with multiple ranks and we encourage users to also
+    run them with multiple ranks when learning about pylops-mpi.
```

Comment:
since you made a new command in the Makefile, makes sense to either add a comment about it here or in the main README under the doc section

Reply:
I have added it over here, I think it would be better as it is for gallery examples....